### PR TITLE
Ch7 updates and edit - take 2

### PR DESCRIPTION
**Pull request by mishaone**
New PR incorporating changes minus Preeti's commits that were included last time.

### CODE REVIEW

File: docs/topics/modules/using_stack_reports_to_analyze_your_application.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
-[id="analytics_stack_report"]
-= Analytics stack report
+[id="using_stack_reports_to_analyze_your_application"]
+= Using stack reports to analyze your application
 
-The {osio} analytics stack report provides comprehensive information regarding your application stack and its component libraries that helps you make informed choices. It analyzes your stack and recommends alternate and additional components to improve your application.
+{osio} includes a stack report feature that analyzes and reports information about a project's application stack and component libraries. This information helps you make informed choices about the code and libraries used in your project.
```
Comment (Preeticp) [inline docs/topics/modules/using_stack_reports_to_analyze_your_application.adoc]: I would suggest, retain 'and recommends alternate and additional components to improve your application.' This is one of the important value adds provided by the stack report and highlighting it upfront would be useful for the user.

File: docs/topics/modules/using_stack_reports_to_analyze_your_application.adoc [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-The stack report is triggered during the *Build Release* stage of the build pipeline. To view the stack report for your quickstart application:
+{osio} generates a stack report for your project when the pipeline build starts the *Build Release* stage of the build pipeline. To view the stack report for your quickstart application:
```
Comment (Preeticp) [inline docs/topics/modules/using_stack_reports_to_analyze_your_application.adoc]: Suggest dropping the words 'build pipeline' at the end of this sentence, it is the third time we mention build and pipelines and seems redundant: {osio} generates a stack report for your project when the pipeline build starts the *Build Release* stage of the build pipeline.

File: docs/topics/modules/analyzing_your_application_stack_dependencies.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+</dependency>
+----
++
+. An error icon (red cross) appears next to the added lines of code. Move the mouse pointer over the icon to view the error message and a recommendation for improving the dependency.
++
+image::red_x.png[Error Icon]
++
+The error indicates that the dependency analyzer has identified a high Common Vulnerabilities and Exposures (CVE) score associated with the current version, 1.1.0. It also recommends using an alternate, secure version, 1.1.2, instead to fix the issue.
+
+. In the code, update the version to `1.2.1` and the error icon disappears.
```
Comment (Preeticp) [inline docs/topics/modules/analyzing_your_application_stack_dependencies.adoc]: The error icon disappearing is the result of the version update, the 'and' in here makes it seem as if it is part of the update action. 
How about:
In the code, update the version to `1.2.1` to see the error icon disappear.
or
In the code, update the version to `1.2.1`, the error icon disappears.

File: docs/topics/modules/using_stack_reports_to_analyze_your_application.adoc (hunk 1 above)
Comment (Preeticp) [reply docs/topics/modules/using_stack_reports_to_analyze_your_application.adoc]: Please replace 'components' with 'dependencies' in the above sentence.

**Comment by mishaone** [comment]
Implemented feedback (thanks, Preeti!) and now merging once build check succeeds.

**Pull request merged**